workflows/labels: skip for staging-next / haskell-updates / python-updates

wolfgangwalther opened the pull request:
Labeling those PRs is more annoying then useful.

## Things done

(not tested, yet)

- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

wolfgangwalther committed:
workflows/labels: skip for staging-next / haskell-updates / python-updates

Labelling those PRs is more annoying then useful.

sternenseemann commented:
This means we loose the python label on python-updates and the haskell one on haskell-updates, though? We should probably figure out whether we can preserve them.

wolfgangwalther committed:
labels: add workflow related labels

This adds some labels related to the higher level workflows of
developing nixpkgs:
- "4.workflow: backport" can be used to filter everything targeting the
stable branches, both release-XX.YY or staging-XX.YY.
- "4.workflow: package set updates" can be used to filter the
haskell/python package set PRs. Can be combined with "6.topic: haskell"
or "6.topic: python".
- "4.workflow: staging" can be used to filter for all staging-next PRs,
both to master and release branches.
